app/routes/companyReview.js

```diff
@@ -0,0 +1,3 @@
+const crudRouter = require('./crudRouter');
+
+module.exports = crudRouter('companyReview', 'company-reviews');
```

Comment:
Please use the following paths for reviews:
`POST /api/companies/<company_id>/reviews` -> create review
`GET /api/companies/<company_id>/reviews` -> get all company reviews
`GET /api/companies/<company_id>/reviews/<review_id>` -> get one review